hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestSchemaOneBackwardsCompatibility.java

```diff
@@ -279,6 +288,8 @@ public void testDelete() throws Exception {
               refCountedDB.getStore().getMetadataTable();
       assertEquals(expectedRegularBlocks + expectedDeletingBlocks,
               (long)metadataTable.get(OzoneConsts.BLOCK_COUNT));
+    //} catch(IOException ex) {
```

Comment:
Do we need this commented out test? Or can we just remove this part?

Reply:
unnecessary, removed.